.ci/test_r_package_windows.ps1

```diff
@@ -65,8 +65,8 @@ if ($env:R_MAJOR_VERSION -eq "3") {
   $RTOOLS_INSTALL_PATH = "C:\rtools40"
   $env:RTOOLS_BIN = "$RTOOLS_INSTALL_PATH\usr\bin"
   $env:RTOOLS_MINGW_BIN = "$RTOOLS_INSTALL_PATH\mingw64\bin"
-  $env:RTOOLS_EXE_FILE = "rtools40-x86_64.exe"
-  $env:R_WINDOWS_VERSION = "4.0.5"
+  $env:RTOOLS_EXE_FILE = "rtools40v2-x86_64.exe"
```

Comment:
Yes these inconsistencies make it difficult sometimes to automate things in R haha. I looked too and have no idea why a `v2` rtools was built, instead of just versioning it the same as R.